supabase/migrations/20230823005207_sessions_table.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE sessions (
+  key VARCHAR(255) PRIMARY KEY,
```

Comment:
I think we are storing 20 characters here always. so maybe something like `char(20)`?

Reply:
Fixed here https://github.com/ubiquity/telegram-ubiquibot/pull/17/commits/721280ce49bdc5dd98bd490ac64fddce293e544f